src/journal-remote/journal-remote-main.c

```diff
@@ -841,31 +841,31 @@ static int parse_argv(int argc, char *argv[]) {
                 case ARG_URL:
                         if (arg_url)
                                 return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
-                                                       "cannot currently set more than one --url");
+                                                       "Cannot currently set more than one --url");
```

Comment:
if you touch these, please also change `--url` to `--url=`, as we usually do when referencing cmdline options that take arguments in our docs and log messages. similar for the other log lines you are touching here.